graphrag_sdk/chat_session.py

```
@@ -182,30 +182,162 @@ def send_message_stream(self, message: str) -> Iterator[str]:
+        >>> from graphrag_sdk import KnowledgeGraph, Orchestrator
+        >>> from graphrag_sdk.ontology import Ontology
+        >>> from graphrag_sdk.model_config import KnowledgeGraphModelConfig
+        >>> model_config = KnowledgeGraphModelConfig.with_model(model)
+        >>> kg = KnowledgeGraph("test_kg", model_config, ontology)
+        >>> cypher_session = CypherSession(model_config, ontology, kg.graph, 
+        ...                               cypher_system_instruction, cypher_gen_prompt, cypher_gen_prompt_history)
+        >>> result = cypher_session.search("What is the capital of France?")
+    """
+
+    def __init__(self, model_config: KnowledgeGraphModelConfig, ontology: Ontology, graph: Graph,
+                cypher_system_instruction: str, cypher_gen_prompt: str, cypher_gen_prompt_history: str):
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)